src/main/java/Duke.java

```diff
@@ -1,10 +1,110 @@
+import java.util.ArrayList;
+import java.util.Locale;
+import java.util.Scanner;
+
 public class Duke {
+
+    private static final String LOGO = " ____        _        \n"
+                                    + "|  _ \\ _   _| | _____ \n"
+                                    + "| | | | | | | |/ / _ \\\n"
+                                    + "| |_| | |_| |   <  __/\n"
+                                    + "|____/ \\__,_|_|\\_\\___|\n";
+    private static final String LINE_SEPARATOR = "------------------------------------";
+    private static final String HELLO_MESSAGE = "Hello! I'm Duke\nWhat can I do for you?";
+    private static final String BYE_MESSAGE = "Bye. Hope to see you again soon!";
+    private static final String ERROR_MESSAGE = "You need to specify the task type!";
+
+    private static ArrayList<Task> tasks = new ArrayList<>();
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        printHelloMessage();
+        handleCommand();
+        printByeMessage();
+    }
+
+    private static void handleCommand() {
+        String input;
+        Scanner in = new Scanner(System.in);
+        input = in.nextLine();
+        while (!input.equals("bye")) {
+            String keyword = input.split(" ")[0].toLowerCase();
+            switch (keyword) {
+            case "bye":
+                continue;
```

Comment:
Thanks for pointing out! I believe this statement is still required since it will break me out of the current loop and bring me to the next loop (which then checks whether my input is "bye" and exits the whole function).